### DISCUSSION TITLE
Gracefull shutdown/restart procedure for whole kubernetes environment using longhorn

### DISCUSSION THREAD

**Discussion by peedy2495** [longhorn/longhorn#4036] [Q&A]
First, I googled a while for this question, but found nothing.
Nor in the official documentation, neither in any Posts.

So, which steps are considered to keep a consistent storage ?
Anything like ceph: `ceph osd set noout; ceph osd set nobackfill; ceph osd set norecover; etc. ...` on every ceph-osd ?

**Comment by PhanLe1010** [comment]
shutdown/restart procedure for whole Kubernetes: Scale down the workload and make sure that all volumes are detached then turn off the cluster
